### ISSUE TITLE
Hololens Windows UWP App Icon specifications missing from  existing documentation

### ISSUE THREAD

**Issue by tank-t-bird** [microsoft/MixedRealityToolkit-Unity#7821]
## Describe the issue
When adding the icons into the Player settings in Unity, it is not clear what size icons should be for the Hololens (1 or 2, doesn't matter). Considering that a Hololens app (for now) is rarely "cross platform", it doesn't seem to make sense that one would need all the icons in all the sizes requested. 

In addition to this, the region right below the "Icon" settings in Unity says the following: "Not applicable for this platform".

If a developer adds every possible icon shape and scale possible, it is also not clear that using a PNG with an alpha will result in the Windows blue color background being added as a default. It is also not clear if the developer should use white for the icon symbol as a default recommendation or if white is specifically reserved for Microsoft apps.

## Feature area
Current documentation in MRTK has no information about the Build/App icons whatsoever. The Unity Documentation has general UWP Windows App Icon information but lacks details specific to the Hololens devices. The Windows UWP App icons documentation seems to focus on Desktop apps and also lacks details for Hololens apps.

## Existing doc link
[Unity UWP App Icons](https://docs.unity3d.com/Manual/class-PlayerSettingsWSA.html#Icon)
[Windows UWP App Icons](https://docs.microsoft.com/en-us/windows/uwp/design/style/app-icons-and-logos)

**Comment by tank-t-bird** [comment]
After I created this issue I found this page [Mixed Reality App Packaging](https://docs.microsoft.com/en-us/windows/mixed-reality/submitting-an-app-to-the-microsoft-store) which gets closer to what I would expect in terms of documentation. For whatever reason it is not in the first search pages of a well known search engine. I still believe it can be improved, in particular the Table section "Where is this displayed" (N/A). Also it doesn't mention the color topics i mentioned in the issue.
